MediaBrowser.WebDashboard/dashboard-ui/strings/ar.json

```diff
@@ -1010,6 +1010,12 @@
   "LabelValue": "القيمة:",
   "LabelVersionInstalled": "{0} مثبتة",
   "LabelVersionNumber": "الإصدار {0}",
+  "LabelServerVersionNumber": "Server Version: {0}",
+  "LabelBuildRevision": "Revision: {0}",
+  "LabelBuildBranch": "Branch: {0}",
+  "LabelBuildCommit": "Commit: {0}",
+  "LabelBuildTagDescription": "Tag: {0}",
+  "LabelProductName": "Name: {0}",
   "LabelVersionUpToDate": "محدث إلى الإصدار الأخير!",
   "LabelVideoCodec": "مقطع الفيديو: {0}",
```

Comment:
This too, is it something rtl-caused?

Reply:
It seems so. Someone with language knowledge will have to have a look.

Reply:
Let's wait for some Arabic-speaking volunteer then.